Directory.Build.targets

```diff
@@ -54,24 +54,8 @@
     <PackageDescription Condition="'$(PackageDescription)' == '' and '$(UseRuntimePackageDisclaimer)' == 'true'">$(RuntimePackageDisclaimer)</PackageDescription>
     <!-- Keep in sync as required by the Packaging SDK in Arcade. -->
     <Description>$(PackageDescription)</Description>
-    <GenerateNuspecDependsOn>ErrorForMissingPackageDescription;$(GenerateNuspecDependsOn)</GenerateNuspecDependsOn>
   </PropertyGroup>
 
-  <!-- Remove when https://github.com/NuGet/Home/issues/10405 is implemented and consumed. -->
-  <Target Name="ErrorForMissingPackageDescription"
```

Comment:
Also removing this block as it didn't work anymore for the NuGet pack as PackageDescription is always set to a default value.